manifests/common/nova.pp

```diff
@@ -13,15 +13,15 @@
   $controller_management_address = $::openstack::config::controller_address_management
 
   class { '::nova':
-    sql_connection     => $::openstack::resources::connectors::nova,
-    glance_api_servers => join($::openstack::config::glance_api_servers, ','),
-    memcached_servers  => ["${controller_management_address}:11211"],
-    rabbit_hosts       => $::openstack::config::rabbitmq_hosts,
-    rabbit_userid      => $::openstack::config::rabbitmq_user,
-    rabbit_password    => $::openstack::config::rabbitmq_password,
-    debug              => $::openstack::config::debug,
-    verbose            => $::openstack::config::verbose,
-    mysql_module       => '2.2',
+    database_connection      => $::openstack::resources::connectors::nova,
+    glance_api_servers       => join($::openstack::config::glance_api_servers, ','),
```

Comment:
These should use spaces instead of tabs and the arrow should start one space from the longest parameter.